packages/test/test-end-to-end-tests/package.json

```diff
@@ -149,27 +149,8 @@
 	},
 	"fluidBuild": {
 		"tasks": {
-			"compile": {
-				"dependsOn": [
-					"...",
-					"^build:esnext"
-				],
-				"script": false
-			},
-			"ci:build": {
-				"dependsOn": [
-					"...",
-					"^build:esnext"
-				],
-				"script": false
-			},
-			"build:test": [
-				"...",
-				"^build:esnext"
-			],
```

Comment:
Unsure if this might still be necessary... unless we update build:test to run build:test:esm as a fluid build task, not just as a npm script, then running build:test as a fluid build task would not ensure dependencies product builds are there, I think?

Reply:
`build:test` does run `build:test:esm`. fluid build recognizes `npm run <script entry>` and populates the task dependency tree with it.